src/client.rs

```diff
@@ -60,9 +62,17 @@ fn client_loop(config: ValidatorConfig) {
     } = config;
 
     loop {
+        let term = Arc::clone(should_term);
```

Comment:
Shouldn't this also work, without the cloning here?

Reply:
You're right - this was a remnant of how the code was structured prior to applying the recommendations from the linter. Fixed this now.